classes/network/Connection.py

```diff
@@ -17,8 +18,12 @@
 from .versions.v578 import v1_15_2, v1_15_2_Input
 from classes.utils.Config import ConfigParser
 
+
+from classes.plugins import event
 config = ConfigParser.load_config(1)
 
+event.registerEvent("chat")
```

Comment:
the positioning looks rather arbitrary for me

Reply:
it is at the top of the file so the event *should* be registered before all functions, but currently it appears python3.8 is having a problem with that (possible ideas to fix in latest comment)

Reply:
what I mean with this is that I believe that this results in spaghetti code if this pattern were to be reproduced over a few more files